cockpit-frontend/src/components/header/Header.jsx

```diff
@@ -21,6 +26,16 @@ export default function Header() {
             Cockpit
           </IconButton>
           <MvpMenu />
+          <Button
```

Comment:
Maybe we can add grid for the items on the app bar to make it responsive when navigator size changes, we can look at it together tomorrow.